### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by mdubus** [meilisearch/instant-meilisearch#337]
The documentation isn't really clear if we want to call some specific routes and not only get results.
Combined with [this issue](https://github.com/meilisearch/meilisearch-js/issues/834), it makes it really hard to understand how it works and what we need to do in some specific use cases.

This could be an example to add to the documentation : 

```
import { instantMeiliSearch } from "@meilisearch/instant-meilisearch";

const searchClient = instantMeiliSearch(
  "https://demos.meilisearch.com",
  PRIVATE_API_KEY
);

 const getIndexList = async () => {
    const indexes = await searchClient.client.listIndexes();
    console.log(indexes);
 };
```

**Comment by curquiza** [comment]
Related to #163

**Comment by bidoubiwa** [comment]
fixed by: #417
